sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.0-beta.10 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.0.0-beta.10 (2024-10-10)
 
 ### Bugs Fixed
 
-### Other Changes
+- [[#6064]](https://github.com/Azure/azure-sdk-for-cpp/issues/6064) Utilize new telemetry features from Azure Core.
```

Comment:
I don't see this a bug fix, nor something that's relevant to the end user. They will not see any behavioral change here, since we are just moving namespaces for an internal implementation.

Reply:
What change do you propose to have listed in the changelog then? There isn't going to be anything else.
And more generally, when we have only the changes that are not immediately visible, what do you think our approach to changelog entries should be? I think this is appropriate.

Reply:
For now, no changes (and hold off on shipping out of band).

We should ship updates to our packages that have meaningful impact or benefit to our end customers, or we are gathering beta feedback of features/designs. Both of which naturally warrant changelog entries that are relevant to the user.

If we have anything else planned that would motivate shipping an update to eventhubs, in the future, we could add those entries at that time.

I'd defer to @RickWinter for whether we need to ship such changes, which a changelog update like this is predicated on.